### DISCUSSION TITLE
How can I use/customize the search feature of the kotlin-toolkit?

### DISCUSSION THREAD

**Discussion by AnkitVerma8584** [readium/kotlin-toolkit#315] [Q&A]
I am using folioReader for reading epubs , but we want to customize only the search feature in the kotlin-toolkit. How to customize the search module and use it on folioReader?

**Comment by mickael-menu** [comment]
Folio Reader is using a really old version of Readium and its own search implementation. I don't think you could easily port the current Readium search service into Folio Reader.

IMHO, your best course of action would be to drop Folio Reader and integrate the latest version of the Kotlin Toolkit in your app.
